src/Payment.ts

```diff
@@ -98,11 +100,15 @@ export class Payment {
     value: number | string,
     options: PaymentOptions = {}
   ): Promise<TxObject> {
+    const { gasLimit, gasPrice } = options
     return this._transaction.prepValueTx(
       this._user.address,
       receiverAddress,
       this._utils.calcRaw(value, 18),
```

Comment:
As this will be a BigNumber after the change, we have to call convertToHex here then

Reply:
Same in Trustline.js